channeldb/channel.go

```diff
@@ -66,6 +66,11 @@ var (
 	// party.
 	chanCommitmentKey = []byte("chan-commitment-key")
 
+	// danglingRemoteUpdatesKey is an entry in the channel bucket that
```

Comment:
Added more explanation, which indeed is what the commit deserves. If further clarification is needed, I'd rather add that as comments to the code because of the better visibility.